webpack/assets/javascripts/compute_resource/ovirt.js

```diff
@@ -34,16 +34,7 @@ export function templateSelected(item) {
       // As Instance Type values will take precence over templates values,
       // we don't update memory/cores values if  instance type is already selected
       if (!$('#host_compute_attributes_instance_type').val()) {
-        const memoryInputElement = document
-          .getElementById('memory-input')
-          .getElementsByTagName('foreman-react-component')[0];
-        memoryInputElement.setAttribute(
-          'data-props',
-          JSON.stringify({
-            ...memoryInputElement.reactProps,
-            defaultValue: result.memory / megabyte,
-          })
-        );
+        setMemoryInputProps({ value: result.memory / megabyte });
```

Comment:
how about storing it as a helper at the top of the file, and using it also on [line 81](https://github.com/theforeman/foreman/pull/8413/files#diff-df73d605ed9a42fc6699f17d7ad6ca641a402d9dd709db645e0aa4eec8907b62R81), e.g:
`const setMemoryValue = () => setMemoryInputProps({ value: result.memory / megabyte });`

Reply:
Sure, why not :) I think more like:
`const setMemoryValue = memoryBytes => setMemoryInputProps({ value: memoryBytes / megabyte });`

Reply:
I've changed the component so it accepts value in bytes, so we don't need the calculation, so the wrapper seemed not so useful.

Reply:
Awesome!